test/parallel/test-gc-http-client-connaborted.js

```diff
@@ -48,11 +48,19 @@ function ongc() {
   countGC++;
 }
 
-setInterval(status, 100).unref();
+setImmediate(status);
 
 function status() {
-  global.gc();
-  console.log('Done: %d/%d', done, todo);
-  console.log('Collected: %d/%d', countGC, count);
-  if (countGC === todo) server.close();
+  if (done > 0) {
+    createClients = false;
+    global.gc();
+    console.log(`done/collected/total: ${done}/${countGC}/${count}`);
+    if (countGC === count) {
+      server.close();
+    } else {
+      setImmediate(status);
+    }
+  } else {
+    setImmediate(status);
+  }
```

Comment:
nit
```suggestion
      return;
    }
  }
  setImmediate(status);
```

Reply:
This is only a cosmetic change that is inconsistent with the original refactor in https://github.com/nodejs/node/commit/47ecf2060343. I would prefer to not apply it.

Reply:
Sure, feel free to ignore and land as is, LGTM anyway.

Reply:
I'll apply it, but it's a little annoying to rerun CI for something like this.